tests/sgl/device/test_persistent_cache.cpp

```
@@ -0,0 +1,292 @@
+            );
+        }
+    );
+    prepare_started_future.wait();
+
+    auto flush_future = std::async(
+        std::launch::async,
+        [&]
+        {
+            writer->flush();
+        }
+    );
```

Cppcheck (2.21.0)

[style] The function 'unused' is never used.

(unusedFunction)